infrastructure/helm-chart/values.yaml

```diff
@@ -8,3 +8,4 @@ global:
     webhooksHost: "webhooks.airy"
     uiHost: "ui.airy"
     chatpluginHost: "chatplugin.airy"
+    toolsHost: "tools.airy"
```

Comment:
Akhq will be deployed and accessible through akhq.{toolsHost} or `akhq.tools.airy`.
I didn't add the DNS entry for the Vagrantbox, as `/etc/hosts` doesn't accept wildcard certificates.

Open to suggestions for this, maybe one option would be to do `tools.airy/akhq`, but this doesn't work out of the box and some tweak would be needed for traefik to strip the `/akhq` prefix, before the request is sent to the app.